scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -1161,13 +1163,42 @@
 (meters/defmeter [cook-mesos scheduler mesos-error])
 (meters/defmeter [cook-mesos scheduler offer-chan-full-error])
 
+(def dummy-fitness-calculator
```

Comment:
If this is used only for testing purposes, we should it move it to a class in the test package.